setup/stack/states/php7.2/php/init.sls

```diff
@@ -14,108 +14,49 @@ php_repo_key:
 
 php7.2:
   pkg.installed:
-    - refresh: true
     - pkgs:
       - libapache2-mod-php7.2
+      - php-json
+      - php7.2-apcu
+      - php7.2-bcmath
+      - php7.2-cli
       - php7.2-curl
       - php7.2-gd
-      - php7.2-json
+      - php7.2-imap
+      - php7.2-mbstring
       - php7.2-mysql
+      - php7.2-opcache
       - php7.2-readline
-      - php7.2-xsl
-      - php7.2-cli
-      - php7.2-zip
-      - php7.2-mbstring
       - php7.2-soap
-      - php7.2-bcmath
-      - php7.2-imap
+      - php7.2-xml
       - php7.2-xmlrpc
+      - php7.2-xsl
+      - php7.2-zip
 
-php7.0:
-  pkg.removed:
-    - pkgs:
-      - libapache2-mod-php7.0
-      - php7.0-curl
-      - php7.0-gd
-      - php7.0-json
-      - php7.0-mysql
-      - php7.0-readline
-      - php7.0-xsl
-      - php7.0-cli
-      - php7.0-zip
-      - php7.0-mbstring
-      - php7.0-xml
-      - php7.0-soap
-      - php7.0-bcmath
-      - php7.0-imap
-      - php7.0-xmlrpc
-
-php7.1:
-  pkg.removed:
-    - pkgs:
-      - libapache2-mod-php7.1
-      - php7.1-curl
-      - php7.1-gd
-      - php7.1-json
-      - php7.1-mysql
-      - php7.1-readline
-      - php7.1-xsl
-      - php7.1-cli
-      - php7.1-zip
-      - php7.1-mbstring
-      - php7.1-soap
-      - php7.1-bcmath
-      - php7.1-imap
-      - php7.1-xmlrpc
-
-php7.3:
-  pkg.removed:
-    - pkgs:
-      - libapache2-mod-php7.3
-      - php7.3-curl
-      - php7.3-gd
-      - php7.3-json
-      - php7.3-mysql
-      - php7.3-readline
-      - php7.3-xsl
-      - php7.3-cli
-      - php7.3-zip
-      - php7.3-mbstring
-      - php7.3-soap
-      - php7.3-bcmath
-      - php7.3-imap
-      - php7.3-xmlrpc
-
-php7.4:
-  pkg.removed:
-    - pkgs:
-      - libapache2-mod-php7.4
-      - php7.4-curl
-      - php7.4-gd
-      - php7.4-json
-      - php7.4-mysql
-      - php7.4-readline
-      - php7.4-xsl
-      - php7.4-cli
-      - php7.4-zip
-      - php7.4-mbstring
-      - php7.4-soap
-      - php7.4-bcmath
-      - php7.4-imap
-      - php7.4-xmlrpc
-
-php8.0:
-  pkg.removed:
+{% for version in ['7.1','7.3','7.4','8.0','8.1'] %}
```

Comment:
Did you include 7.1 here for downward compatibility with existing instances? And if so, what about 7.0?